api/src/model/request/play_asset_delivery_request.gd

```diff
@@ -20,7 +20,7 @@
 # signals emitted from PlayAssetDelivery Android Plugin.
 #
 # ##############################################################################
-class_name PlayAssetPackRequest
+class_name PlayAssetDeliveryRequest
```

Comment:
I renamed the base class to PlayAssetPackRequest since we will be having a subclass named PlayAssetPackRequest (Used for ```request_asset_pack()``` and ```get_asset_pack_state()```).